SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -37,7 +38,7 @@ DECLARE_CYCLE_STAT(TEXT("UpdateSpatialPosition"), STAT_SpatialActorChannelUpdate
 DECLARE_CYCLE_STAT(TEXT("ReplicateSubobject"), STAT_SpatialActorChannelReplicateSubobject, STATGROUP_SpatialNet);
 DECLARE_CYCLE_STAT(TEXT("ServerProcessOwnershipChange"), STAT_ServerProcessOwnershipChange, STATGROUP_SpatialNet);
 DECLARE_CYCLE_STAT(TEXT("ClientProcessOwnershipChange"), STAT_ClientProcessOwnershipChange, STATGROUP_SpatialNet);
-DECLARE_CYCLE_STAT(TEXT("GetOwnerWorkerAttribute"), STAT_GetOwnerWorkerAttribute, STATGROUP_SpatialNet);
+DECLARE_CYCLE_STAT(TEXT("GetOwnerWorkerAttribute"), STAT_GetOwningClientWorkerId, STATGROUP_SpatialNet);
```

Comment:
Is the OwnerWorkerAttribute always going to be a client? Never a server worker?

Reply:
Ah, yea, looks like it's used by both, in a sense. Servers have one USpatialNetConnection per PlayerController they have knowledge of. Clients have a single USpatialNetConnection representing _the server side_.  It looks like only the USpatialNetConnections representing clients actually have this set though (hence my initial assuming this was a simple rename). 

I'm still of the opinion it's better to have the explicit naming for clarity, however I might change it to an option so it's more explicit that it's unset on client workers?

Reply:
If it's only ever set if there's a possession ongoing, you could name it PossessingClientWorkerId and make it an option and I think it would be clear when it would and would not be set.

Reply:
Afaik, in Unreal, the `possession` naming refers only to Actors whereas connections are `owned`. Happy to do the option choice, but I think it should still be `OwningClientWorkerId`?